docs/triage role expectations.md

```diff
@@ -0,0 +1,100 @@
+# Triage Role Expectations
+
+Users with Triage-level access are selected contributors who can proactively label/triage issues and pull requests 
+without write access to the Archipelago repository.
+
+Triage users are not necessarily official members of the Archipelago organization, for the list of core maintainers,
+please reference [ArchipelagoMW Members](https://github.com/orgs/ArchipelagoMW/people) page.
+
+## Access Permissions
+
+Triage users have the following permissions:
+
+* Apply/dismiss labels on all issues and pull requests.
+* Close, reopen, and assign all issues and pull requests.
+* Mark issues and pull requests as duplicate.
+* Request pull request reviews from repository members.
+* Hide comments in issues or pull requests from public view.
+    * Hidden comments are not deleted and can be reversed by another triage user or repository member with write access.
+* And all other standard permissions granted to regular GitHub users.
+
+For more details on permissions granted by the Triage role, see 
+[GitHub's Role Documentation](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization).
+
+## Expectations
+
+Users with triage-level permissions have no expectation to review code, but, if desired, to review pull requests/issues 
+and apply the relevant labels and ping any relevant [code owners](./CODEOWNERS) for review. Triage users are also not 
```

Comment:
If world maintainers are being labeled as collaborators (ie being granted read or triage permissions) this should be changed to "request reviews from" instead of ping